settings.gradle

```diff
@@ -1,3 +1,33 @@
+import org.apache.tools.ant.DirectoryScanner
+
+DirectoryScanner.removeDefaultExclude('**/#*#')
+DirectoryScanner.removeDefaultExclude('**/%*%')
+DirectoryScanner.removeDefaultExclude('**/*~')
+DirectoryScanner.removeDefaultExclude('**/.#*')
+DirectoryScanner.removeDefaultExclude('**/.DS_Store')
+DirectoryScanner.removeDefaultExclude('**/._*')
+DirectoryScanner.removeDefaultExclude('**/.bzr')
+DirectoryScanner.removeDefaultExclude('**/.bzr/**')
+DirectoryScanner.removeDefaultExclude('**/.bzrignore')
+DirectoryScanner.removeDefaultExclude('**/.gitignore')
+DirectoryScanner.removeDefaultExclude('**/.cvsignore')
+DirectoryScanner.removeDefaultExclude('**/.git')
+DirectoryScanner.removeDefaultExclude('**/.git/**')
+DirectoryScanner.removeDefaultExclude('**/.gitattributes')
+DirectoryScanner.removeDefaultExclude('**/.gitmodules')
+DirectoryScanner.removeDefaultExclude('**/.hg')
+DirectoryScanner.removeDefaultExclude('**/.hg/**')
+DirectoryScanner.removeDefaultExclude('**/.hgignore')
+DirectoryScanner.removeDefaultExclude('**/.hgsub')
+DirectoryScanner.removeDefaultExclude('**/.hgsubstate')
+DirectoryScanner.removeDefaultExclude('**/.hgtags')
+DirectoryScanner.removeDefaultExclude('**/.svn')
+DirectoryScanner.removeDefaultExclude('**/.svn/**')
+DirectoryScanner.removeDefaultExclude('**/CVS')
+DirectoryScanner.removeDefaultExclude('**/CVS/**')
+DirectoryScanner.removeDefaultExclude('**/SCCS')
+DirectoryScanner.removeDefaultExclude('**/SCCS/**')
+DirectoryScanner.removeDefaultExclude('**/vssver.scc')
```

Comment:
any reason for this change?

Reply:
I was getting error while building the project and the only solution which worked is the above one for me.